routes/course.js

```diff
@@ -112,6 +112,14 @@ router.post("/api/courses", auth, async (req, res) => {
           message: "Invalid combination of plan_id and distribution_ids.",
         });
     });
+    const retrievedCourses = await Courses.findByPlanId(course.plan_id); 
+    for (const existingCourse of retrievedCourses) {
+      if(existingCourse.number === course.number && existingCourse.term === course.term && existingCourse.year === course.year && existingCourse.title === course.title){
```

Comment:
wouldn't it suffice to check for number and title? why are we checking for term and year? 

If student took course in Fall 2021, should they be able to take it in Spring 2021? Oh I guess the student might be retaking a course. 

This is a genuine question btw, I'm not sure what the answer is. :))

Reply:
you can definitely reason both ways, pls lmk what you think!

Reply:
Yup, that's what we were thinking. In case someone needs to retake the class, or perhaps they drop the class and don't want to go back and update their old semester's plan. Happy to switch to just title and number instead tho!

Reply:
nah you're right! i like this.